main/CoffeeMaker/CoffeeMakerApp.cpp

```diff
@@ -169,6 +169,7 @@ void CoffeeMakerJsonParser(CoffeeMakerJson_str *CoffeeMakerJson, char *CoffeeMak
  */
 void ApplyOnScreen(CoffeeMakerJson_str *CoffeeMakerJson)
 {
+    ESP_LOGE(TAG, "apply on screen");
```

Comment:
is this log needed?

Reply:
It was for my test, now I can remove it .